package.json

```diff
@@ -23,6 +23,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.57",
```

Comment:
Adding material lab for [Skeletons](https://material-ui.com/components/skeleton/)